azure-pipelines.yml

```diff
@@ -100,15 +100,6 @@ stages:
             arguments: 'locals all -clear'
         - script: eng\common\cibuild.cmd -configuration $(_BuildConfig) -prepareMachine $(_BuildArgs)
           displayName: Build and Publish
-        - task: PublishBuildArtifacts@1
-          displayName: Upload TestResults
-          condition: always()
-          continueOnError: true
-          inputs:
-            pathtoPublish: artifacts/TestResults/$(_BuildConfig)/
-            artifactName: $(Agent.Os)_$(Agent.JobName) TestResults
-            artifactType: Container
-            parallel: true

```

Comment:
To also clean up the new warning in builds, change the `enablePublishTestResults` parameter [above](https://github.com/dotnet/blazor/pull/2049/files#diff-fec826feae04e51c0d94076385408bdcR44) to `false`

Reply:
Thanks 😄. Updated.